common/theme.c

```diff
@@ -49,20 +67,19 @@ void themeStartup(ThemePreset preset) {
     char tmp_path[PATH_MAX] = {0};
 
     #ifdef __SWITCH__
-    tmp_path[0] = '/';
+    tmp_path[0] = '/';/*will this work on the windows version?*/
```

Comment:
Yes it will. A common misconception is that Windows only accepts \. That is false, Windows accepts both \ and /.